docs/analytics_welcome/03_how_we_work.md

```diff
@@ -1,21 +1,74 @@
 # How We Work (WIP)
+
 ## Meetings
-### Week 1
+
+### Existing Meetings
+
+#### Week 1
+
 (week-one-meeting)=
-#### Technical Onboarding - Analytics Tools Introduction
-### Recurring
+##### Analytics Introduction + Technical Onboarding (1 hour)
+
+#### Recurring
+
 (weekly-data-services)=
-#### Data Services All-Hands
+##### Data Services All-Hands (Weekly, 1 hour)
+
 (daily-stand-ups)=
-#### Daily Analyst Standups
+##### Analyst Standups (Tuesday - Friday, 15 minutes)
+
+(lunch-and-learn)=
+##### Lunch & Learn (Weekly, TBD)
+
+### Meeting Standards
+
+#### Meeting Preparation
+
+(slack-intro)=
 ## Communication Channels
```

Comment:
Can we turn this section into a table?

E.g.

| channel | purpose |
| -------- | -------- |
| #data-analysis | discuss ... |
| #data-office-hours | something .. |
| #data-analysis-notify | etc.. |

Let's do short and complete sections to stubbing out many nested sections.

Reply:
collapsed nested headers in slack channel section to tables like the example above